app/views/user_mentee_applications/_table_row.html.erb

```diff
@@ -8,16 +8,17 @@
   <% end %>
 
   <%= render TableCellComponent.new(container_link: mentee_application) do %>
-    <%= mentee_application.country %>
+    <%#= mentee_application.notes %>
   <% end %>
-
+  
   <%= render TableCellComponent.new(container_link: mentee_application) do %>
-    <%#= mentee_application.notes %>
+    <%= mentee_application.current_status %>
```

Comment:
Ultra small thing, but you can call `humanize` on this to make it look a bit more natural on the output. This being an admin-facing feature, I don't think it's as important to eliminate all code-ish looking stuff though.
```rb
mentee_application.current_status #=> "stage_one"
mentee_application.current_status.humanize #=> "Stage one"
```